draft-barnes-acme.md

```diff
@@ -1459,14 +1450,131 @@ TODO
 
 # Security Considerations
 
-TODO
+ACME is a protocol for managing certificate that attest to identifier/key bindings.  Thus the foremost security goal of ACME is to ensure the integrity of this process.  ACME identifies clients by their account keys, so this overall goal breaks down into two more precise goals:
+
+1. Only an entity that controls a identifier can get an account key authorized for that identifier
+2. A certificate can only be managed by the holder of an account key authorized for the identifiers in the certifcate
```

Comment:
Well, "manage" means "[do whatever ACME lets you do with a cert](https://youtu.be/714-Ioa4XQw?t=12)".  Right now that means "issue", "revoke", and "renew" (with the last one being more open).  But you could envision other things like "change the set of names in the certificate".